### PR TITLE
SOLACE: Add meet-team page

### PR DESCRIPTION
**Pull request by pankaj3108**
Add meet-team page

### CODE REVIEW

File: src/components/TeamPage/MyTeam.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <div className="meetTeam-parent">
+      <div className="meetTeam">
+        <p className="meetTheTeam">Meet the Team</p>
+        <p className="meetTheTeam-other">Meet our team of professionals to serve you</p>
+      </div>
+      <div className="group">
+        {ProfileDatas.map((item: ProfileData, idx: number) => {
+          return (
+            <div className="overlap-group">
+            {/* //   <div className="text-wrapper-2">{item.name}</div>
```
Comment (rishup132) [inline src/components/TeamPage/MyTeam.tsx:19]: Remove these.

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -24,37 +23,35 @@ const Footer: React.FC = () => {
...
+                    <img src={vertical} alt="logo"/>
+                </div>
+                <div className="footer-links">
+                    <ul>
+                        <li>About Us</li>
+                        <li>Pricing</li>
+                        <li>Support</li>
+                        <li>Getting Started</li>
+                        <li>Privacy Policy</li>
+                    </ul>
```
Comment (rishup132) [inline src/components/Footer/Footer.tsx:50]: Let's use list and iterate over it. It will be useful as we will be adding links and other things on it.

Comment (pankaj3108) [reply src/components/Footer/Footer.tsx:50]: Done.

File: src/components/TeamPage/MyTeam.tsx (hunk 1 above)
Comment (pankaj3108) [reply src/components/TeamPage/MyTeam.tsx:19]: Done.

**Review by rishup132 — APPROVED** [review]

**Pull request merged**